### PR TITLE
Error integration

### PR DESCRIPTION
**Pull request by phiden**
This is part of discovery for how to incorporate documetation @melwoodard's writing into the design system -- post-PR will push to gh-pages.

### CODE REVIEW

**Comment by jbalboni** [comment]
@phiden For some reason the files in the dist folder are committed in here (they're normally ignored in the gitignore file). Can you remove them?

**Comment by phiden** [comment]
Yes, let me first merge in your master with the new image path stuff.

File: docs/Messaging and error handling/Error Message Standardization Content Style Guide.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+*Note: The guidelines below do not apply to inline error messages in form fields (ie, “Please enter a valid Social Security number.”). Those will be handled separately.*
+
+## Structure
+
+Every error message will have **up to** 3 parts:
+
+<img src="/dist/img/docs/error_msg_content_guide.png" width="608">
```
Comment (cehsu) [inline docs/Messaging and error handling/Error Message Standardization Content Style Guide.md]: Should we remove "/dist" from this path? As done elsewhere?

Comment (phiden) [reply docs/Messaging and error handling/Error Message Standardization Content Style Guide.md]: Thanks, I knew I'd missed one!

**Pull request merged**